From ce1b7028e18acdfe05bd7dfe650a6d20a24e70db Mon Sep 17 00:00:00 2001
From: Kamal Mostafa <kamal@canonical.com>
Date: Tue, 26 Jun 2012 07:53:58 -0700
Subject: [PATCH] UBUNTU: SAUCE: input: Cypress PS/2 Trackpad set default
 debug_level=0

BugLink: http://launchpad.net/bugs/978807

Set Cypress debug_level to 0, parm psmouse.cy_debug adjusts it.

Signed-off-by: Kamal Mostafa <kamal@canonical.com>
Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---
 drivers/input/mouse/cypress_ps2.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/input/mouse/cypress_ps2.c b/drivers/input/mouse/cypress_ps2.c
index a370b3a..45afe8f 100644
--- a/drivers/input/mouse/cypress_ps2.c
+++ b/drivers/input/mouse/cypress_ps2.c
@@ -22,7 +22,9 @@
 
 #define CYTP_DBG 1
 #if CYTP_DBG
-static int debug_level = 1;
+static int debug_level;
+module_param_named(cy_debug, debug_level, int, 0644);
+MODULE_PARM_DESC(cy_debug, "Set CyPS/2 debug output level (0, 1, or 2)");
 #define cytp_dbg(fmt, ...)  \
 	do {  \
 		if (cytp && debug_level)  \
-- 
1.8.0

